ray-operator/controllers/ray/utils/util.go

```diff
@@ -68,6 +68,27 @@ func IsRunningAndReady(pod *corev1.Pod) bool {
 	return false
 }
 
+func CheckRouteName(ctx context.Context, s string, n string) string {
+	log := ctrl.LoggerFrom(ctx)
+
+	// 6 chars are consumed at the end with "-head-" + 5 generated.
+	// Namespace name will be appended to form: {name}-{namespace} for first host
+	//   segment within route
+	// 63 - (6 + 5) - (length of namespace name + 1)
+	// => 52 - (length of namespace name + 1)
+	// => 51 - (length of namespace name)
+	maxLength := 51 - len(n)
+
+	if len(s) > maxLength {
+		// shorten the name
+		log.Info(fmt.Sprintf("route name is too long: len = %v, we will shorten it to = %v\n", len(s), maxLength))
```

Comment:
the return carriage can be removed as it's formatted by the logger.

Reply:
Removed